packages/wonder-blocks-tooltip/src/components/tooltip-bubble.tsx

```diff
@@ -3,26 +3,9 @@ import * as React from "react";
 import {View} from "@khanacademy/wonder-blocks-core";
 import {color, spacing} from "@khanacademy/wonder-blocks-tokens";
 
-import type {StyleType} from "@khanacademy/wonder-blocks-core";
 import TooltipContent from "./tooltip-content";
 import TooltipTail from "./tooltip-tail";
-
-import type {getRefFn, Offset, Placement} from "../util/types";
-
-export type PopperElementProps = {
```

Comment:
**note:** Moved this to `util/types.ts` (below) as it is used in different files, so it made sense to move it to a more shared place.